sips/minutes/_posts/2017-02-14-sip-minutes.md

```diff
@@ -0,0 +1,246 @@
+
+The current SIP tries to make it behave as expected by the users in common cases.
+
+**Lukas** Let's take a simple example, you have an object with a static field and a non-static field. Now the user code, the program access the non=static field first. That means the module gets initialized, but the static field lives in the companion class, right? So the static field is not initialized necessary even though you access the module. Assuming the field initializers have side effects then you can observe the differences. Then the static field will only be initialized at some point later when you use the class and not when you access the module.
+
+**Dmitry** The idea is that the module should force execution of static initializers of the fields of the companion class. Which means that you won't be able to observe the difference in the order. It will look like all of them initialized at the same time.
+
+**Martin** How can that be done?
+
+**Dmitry** Just refer to the class in bytecode. Just mention the signature
+
+**Martin** Does that mean we have to change the immediate code to enforce that?
+
+**Dmitry** Yeah you need to make one reference. It's very easy to write an expression in Scala which returns a null but ____ (7:45)
+
+**Martin** Sure but we can't do that right now.
+
+**Dmitry** I wouldn't do it.
+
+**Martin** Are there good reasons to do that anyway?
+
+**Dmitry** Maintaining semantics. If we don't do it, you can observe the fact that some fields are initialized, some aren't.
+
+**Martin** But we only have to do it for those companion objects that have both statics and fields. It doesn't leak into binary compatibility.
+
+**Jorge** You mentioned syntactic restrictions. Which kind?
+
+**Dmitry** Static fields should proceed any non-static fields. the fact that static fields can be initialized earlier but not later means that you won't be able to observe the fact ____ (8:40). If we were to allow non static fields to proceed, because static fields can be initialized earlier, you can see ____.
+
+**Sébastien** What about non-field statements?
+
+**Dmitry** They should also be after all the statics.
+
+**Sébastien** That's not written in the SIPs.
+
+**Dmitry** Will update. The last question is whether it will effect binary compatibility. YES! The point is you should be able to call and access stuff which is static through a more efficient way on the JVM. So removing @static annotation will be a binary incompatible change. If we add forwarders, adding static won't be a binary incompatible change because you'll still have fowarders which forward to the static thing. Both methods and fields.
```

Comment:
needs backquotes around `@static` so we aren't @-mentioning that GitHub user. there are other places that need this fix too